### PR TITLE
feat(468): enable cors for ui

### PR DESCRIPTION
**Pull request by minz1027**
Enable cors for UI, so that UI can get the artifact manifest.

Related to https://github.com/screwdriver-cd/screwdriver/issues/468

### CODE REVIEW

**Comment by d2lam** [comment]
@petey  Can you help review this one?

**Comment by FenrirUnbound** [comment]
Is there a way to assert a negative test case where a request is denied because CORS is being properly enforced?

File: bin/server [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
+
 'use strict';
 
+/* eslint-disable import/no-dynamic-require */
```
Comment (petey) [inline bin/server]: Put this next to the `require` that specifically triggers this error, and remember to re-enable it immediately after.

**Comment by petey** [comment]
@FenrirUnbound has a good point that there should be a test that validates that CORS is running one way or another. I think we can rely on hapijs having proper full-coverage tests for CORS implementation, but we need to verify that when configured for CORS this server does return the proper headers.

**Comment by minz1027** [comment]
@petey @FenrirUnbound Added a test in the positive case to validate cors header.

**Pull request merged**